CONTRIBUTING.md

```diff
@@ -38,7 +38,7 @@ The library sources are in: [src/lib](https://github.com/Microsoft/TypeScript/tr
 
 To build the library files, run
 ```Shell
-jake lib
+jake LKG

```

Comment:
no, it is lib that builds the library files.

`jake lib` -> `src\lib\*.d.ts` combine them into `built\local\lib.*.d.ts`
`jake LKG` -> `jake local` with `debug=false`, copy `built\local\*` to `lib\*` in preparation of publishing an npm

so files in `\lib\` should not be touched by hand, only through `jake LKG`